Source/SIMPLib/Plugin/SIMPLPluginConstants.h

```diff
@@ -21,7 +21,7 @@ const QString StatusLog("StatusLog");
 const QString OutputLinks("OutputLinks");
 const QString Message("Message");
 const QString Code("Code");
-const QString FilterHumanLabel("FilterHumanLabel");
+const QString HumanLabel("HumanLabel");
```

Comment:
I've got this strange feeling that this needlessly breaks everyone's pipelines.

Reply:
I'm not sure if it will or not. We cue off of the class name or the UUID when reading pipelines. The Human Label is written into the pipeline to help those that want to edit the pipeline but I don't think it will break anything per se. I am wondering why we needed to change that constant?

Reply:
I guess you're right.  The pipeline json has underscores in it.  I missed that part.

Reply:
I'm going to change it back just in case.